hw_7/task_34.py

```diff
@@ -0,0 +1,40 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в 
+# его стихах ритм. Поскольку разобраться в его кричалках не 
+# настолько просто, насколько легко он их придумывает, Вам стоит
+# написать программу. Винни-Пух считает, что ритм есть, если число 
+# слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то
+# они разделяются дефисами. Фразы отделяются друг от друга пробелами. 
+# Стихотворение  Винни-Пух вбивает в программу с клавиатуры. В ответе 
+# напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”,
+# если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+#     **Вывод:** Парам пам-пам  
+
+phrases = input().split()
+vowels = {"а", "о", "у", "ы", "э", "е", "ё", "и", "ю", "я"}
+flag = True
+
+count_main = 0
+for el in phrases[0]:
+    if el in vowels:
+        count_main += 1
+
+if len(phrases) > 1:
+    for phrase in phrases[1:]:
+        count = 0
+        for el in phrase:
+            if el in vowels:
+                count += 1
+        if count != count_main:
+            flag = False
+            break
+else:
+    if count_main == 0:
+        flag = False 
+
+if flag: print("Парам пам-пам")
+else: print("Пам парам")
```

Comment:
Хоть такая запись и допустима, но в `Python` принято использовать традиционную.
```suggestion
if flag:
    print("Парам пам-пам")
else:
    print("Пам парам")
```

Reply:
Так как это домашнее задание выдается после темы про функции высшего порядка, то здесь рекомендуется использовать именно этот инструмент. Попробуйте решить так.